src/index.js

```diff
@@ -209,10 +214,11 @@ class HCaptcha extends React.Component {
 
       const hcaptcha = this._hcaptcha;
       //Render hCaptcha widget and provide necessary callbacks - hCaptcha
-      const captchaId = hcaptcha.render(this.ref.current, renderParams);
+      const id = hcaptcha.render(this.ref.current, renderParams);
 
-      this.setState({ isRemoved: false, captchaId }, () => {
-        onReady && onReady();
+      this.setState({ isRemoved: false, captchaId: id }, () => {
+        onRender && onRender();
+        onReady  && onReady();
```

Comment:
There are two options here:
- Introduce new prop called onReady
- Use onLoad prop instead

hCaptcha onLoad functionality has always centered around the API initially loading and not on the component being ready to use. However, may developers utilize `onLoad` as part of the component state being ready and adding a reference property to execute the hCaptcha component programatically. 

With that in mind, we can utilize onLoad every time the component is created, but I want to make sure nothing breaks for developers that currently use onLoad for single instance of setup. 

@hCaptcha/react-reviewers please let me know what may be best here.

Reply:
Depending on which we choose will also dictate the version to update to for this PR.

Reply:
* is `onRender` needed here?
* `onReady` should be actually handy, I'd vote for it

Reply:
OnRender is used as a callback in the handleOnLoad function to let the user know that the API has loaded. The other option is to only call `onRender` here and then call onReady in the other areas of the code where `renderCaptcha` is executed.

Reply:
It's interesting because it has been working without it, and it's not related to the actual console error fix
Actually, not adding onRender here is working the same way (from what I see)

(somehow this comment was in Pending)